components/org.wso2.carbon.identity.api.server.branding.preference.management/org.wso2.carbon.identity.api.server.branding.preference.management.v1/src/main/resources/branding-preference.yaml

```diff
@@ -358,4 +581,4 @@ components:
         authorizationCode:
           authorizationUrl: 'https://localhost:9443/oauth2/authorize'
           tokenUrl: 'https://localhost:9443/oauth2/token'
-          scopes: {}
+          scopes: {}
```

Comment:
Shall we add a new line

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/495/commits/f15e2dac78ced6aee72e580d1fad018ea99574c3